pkg/plugins/policies/meshpassthrough/plugin/v1alpha1/testdata/basic.clusters.golden.yaml

```diff
@@ -0,0 +1,134 @@
+resources:
+- name: meshpassthrough_*.example.com_*
+  resource:
+    '@type': type.googleapis.com/envoy.config.cluster.v3.Cluster
+    altStatName: meshpassthrough___example_com__
```

Comment:
should we think about replacing `*` with something else than `_`? it's going to be hard to distinguish it, right?

Reply:
Do you have maybe some idea? The `altStatName` is replaced building cluster/filter chain so it's not replaced by me

Reply:
we could do something like wildcard, so it's `meshpassthrough_wildcard_example_com_wildcard`?